flume-ng-sinks/flume-hdfs-sink/src/test/java/org/apache/flume/sink/hdfs/TestHDFSEventSink.java

```diff
@@ -1323,6 +1323,64 @@ public void testCloseOnIdle() throws IOException, EventDeliveryException, Interr
     fs.close();
   }
 
+  @Test
+  public void testBlockCompressSequenceFileWriterSync() throws IOException, EventDeliveryException, InterruptedException {
```

Comment:
Please also rebase on trunk. We've added a checkstyle plugin to the Maven build that will run and tell you what's wrong. In this case, the line exceeds 100 chars.

A reasonable way for you to build Flume and run your test while invoking checkstyle is:

mvn clean install -DfailIfNoTests=false -Dtest=TestHDFSEventSink